scripts/configure_dev.sh

```diff
@@ -43,38 +49,40 @@ function install_or_upgrade {
 function install_windows_shellcheck() {
     version="v0.7.1"
     if ! wget https://github.com/koalaman/shellcheck/releases/download/$version/shellcheck-$version.zip -O /tmp/shellcheck-$version.zip; then
-        rm /tmp/shellcheck-$version.zip &> /dev/null
+        rm /tmp/shellcheck-$version.zip &>/dev/null
         echo "Error downloading shellcheck $version"
         return 1
     fi
 
     if ! unzip -o /tmp/shellcheck-$version.zip shellcheck-$version.exe -d /tmp; then
-        rm /tmp/shellcheck-$version.zip &> /dev/null
+        rm /tmp/shellcheck-$version.zip &>/dev/null
         echo "Unable to decompress shellcheck $version"
         return 1
     fi
 
     if ! mv -f /tmp/shellcheck-$version.exe /usr/bin/shellcheck.exe; then
-        rm /tmp/shellcheck-$version.zip &> /dev/null
+        rm /tmp/shellcheck-$version.zip &>/dev/null
         echo "Unable to move shellcheck to /usr/bin"
         return 1
     fi
 
-    rm /tmp/shellcheck-$version.zip &> /dev/null
+    rm /tmp/shellcheck-$version.zip &>/dev/null
 
     return 0
 }
 
 if [ "${OS}" = "linux" ]; then
-    if ! which sudo > /dev/null; then
+    if ! which sudo >/dev/null; then
         "$SCRIPTPATH/install_linux_deps.sh"
     else
         sudo "$SCRIPTPATH/install_linux_deps.sh"
     fi
 elif [ "${OS}" = "darwin" ]; then
     if [ "${CIRCLECI}" != "true" ]; then
         brew update
-        brew tap homebrew/cask
+        if [[ $(brew --version | cut -d' ' -f2) < "2.5.0" ]]; then
+            brew tap homebrew/cask
+        fi
```

Comment:
This is the only non-whitespace change.

Reply:
maybe 
```suggestion
        if [[ $(brew --version | cut -d' ' -f2 | head -1) < "2.5.0" ]]; then
            brew tap homebrew/cask
        fi
```

Reply:
I completely agree with you @bbroder-algo , it is useful if brew outputs more than one line indeed! Will make the change right now!

Reply:
```
bob@algobob ~ #brew --version
Homebrew 4.0.24
Homebrew/homebrew-core (git revision 25be650e536; last commit 2023-02-16)
Homebrew/homebrew-cask (git revision f7c0fa4386; last commit 2023-02-16)
bob@algobob ~ #
```

Reply:
great! Thanks! Change is on the way!

Reply:
Done! @winder @bbroder-algo, thank you!

Reply:
And I apologize @winder I think I dismissed something here wrongly instead of my own last message! My bad!